src/sajari.js

```diff
@@ -23,13 +23,32 @@ const makeRequest = (address, body, callback) => {
     if (request.readyState !== 4) return;
 
     if (request.status === 200) {
-      callback(null, JSON.parse(request.responseText));
-    } else {
+      try {
+        callback(null, JSON.parse(request.responseText));
+      } catch (e) {
+        callback("Error parsing response.", null);
+      }
+      return;
+    }
+
+    if (request.status === 403) {
```

Comment:
Also make sure that the actual error is kept around too (i.e. so that you can log it)?